csharp/src/Apache.Arrow.Flight.Sql/CancelFlightInfoRequest.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using Google.Protobuf;
+using Google.Protobuf.Reflection;
+
+namespace Apache.Arrow.Flight.Sql;
+
+public sealed class CancelFlightInfoRequest : IMessage<CancelFlightInfoRequest>
```

Comment:
We shouldn't shadow the proto buf class like this.  If we need to extend it we should use a partial class to add what is needed

Reply:
There's no implementation of `CancelFlightInfoRequest` class thus I need to understand what we have to do with this?
@jeremyosterhoudt

Reply:
done and implemented and tested